src/test/java/org/folio/search/controller/SearchInstanceIT.java

```diff
@@ -105,7 +105,7 @@ void responseContainsOnlyBasicInstanceProperties() throws Exception {
     var response = doSearchByInstances(prepareQuery("id=={value}", getSemanticWebId()))
       .andExpect(jsonPath("totalRecords", is(1)))
       // make sure that no unexpected properties are present
-      .andExpect(jsonPath("instances[0].length()", is(8)));
+      .andExpect(jsonPath("instances[0].length()", is(11)));
```

Comment:
What has caused this expectation to change?

Why would moving to Elastic Search 8 mean the expectations for a API response changes?

Reply:
SearchInstanceIT doesn't use Elasticsearch but MockMvc that is initialized with an instance of a Instance.java: https://github.com/folio-org/mod-search/blob/e7cde64376c467a8fb62688f2c989c28dce5253b/src/test/java/org/folio/search/controller/SearchInstanceIT.java#L39-L42

This pull requests adds
```
"required": ["electronicAccess", "notes", "items", "holdings"]
```
to instance.json making these array properties an empty array by default in Instance.java. Before they were null/omitted by default.

"items" array is not affected because it is an array with two elements in this unit test.

Reply:
Thanks. 

I take that mean that this is a side effect of changing the API definition to change the serialization to include empty arrays for these properties.

Is that correct?

Reply:
Yes.

Reply:
Thank you for clarifying. I cannot resolve this comment, please do so if you can.